### DISCUSSION TITLE
Controlling the display of data points and axes

### DISCUSSION THREAD

**Discussion by andreaswallberg** [gosling-lang/gosling.js#975] [Q&A] [answered]
Dear developers,

I am testing out Gosling and I am **really** impressed. I am using some CSV files of my own and I have a few questions:

* My genomes are from non-model organisms and segments of data often correspond to contigs/scaffolds. The "chromosome" convention with lables such as "chr1" does not fit my use case. Is it possible to overrule this nomenclature and use any name for any sequence? For example, for now, I renamed my sequence to "chr1" just to make be able to more easily work my way through the tool, but that labels makes no sense biologically for my organism.
* How can I show all data points at once, regardless of zoom level?
* How do I set min and max values for the Y-axis?

I am grateful for hints and answers.

Thanks for a great tool!

**Comment by andreaswallberg** [comment]
For the first point, I figured out I could use `"assembly": "unknown",` and for the last point `"domain": [0, 1]`,  such that:

```
      "alignment": "overlay",
	  "width": 1000,
	  "height": 600,
	  "assembly": "unknown",
```

and
```
          "mark": "point",
          "x": {"field": "POS", "type": "genomic", "axis": "bottom"},
          "y": {"field": "FST", "type": "quantitative", "axis": "none", "domain": [0, 1]},
```

But I am lost on the second point above. I am trying to emulate a Manhattan plot and it is essential to show all data points at all zoom levels. How can I ensure data points are not displacing or hiding each other?

**Reply by sehilyi** [reply, marked as answer]
Hi @andreaswallberg, you are right about the first and third answers. Another way for the first point would be defining custom genome assembly as described in https://github.com/gosling-lang/gosling.js/pull/776

Regarding the second point, you can set a sufficiently large number for `sampleLength` when you load the data. You can search the keyword on [this documentation](http://gosling-lang.org/docs/data).
